src/app-shared/core/components/form-control-errors/form-control-errors.component.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Provides a helper component that can be used to display form control errors.
+ */
+import { ChangeDetectionStrategy, Component, Host, Input, OnInit } from '@angular/core';
+import { FormControl, FormGroupDirective } from '@angular/forms';
+import { ValidationErrors } from '@angular/forms/src/directives/validators';
+import { isEmpty, isObject } from 'lodash';
+import { Observable } from 'rxjs';
+import { filter, map, startWith } from 'rxjs/operators';
+
+
+@Component({
+  selector: 'app-form-control-errors',
+  templateUrl: './form-control-errors.component.html',
+  styleUrls: ['./form-control-errors.component.scss'],
+  // TODO: Use `OnPush` change detection.
+  changeDetection: ChangeDetectionStrategy.Default,
+  preserveWhitespaces: false,
+})
+export class FormControlErrorsComponent implements OnInit {
+  public formControl: FormControl;
+  public formControlErrors$: Observable<string[] | null>;
+
+  @Input()
+  public readonly formControlPath: string;
+
+  @Input()
+  public readonly namespace: string;
+
+  /**
+   * A list of `ValidationErrors` that should be handled as trees.
+   */
+  protected readonly multiLevelValidationErrors: string[] = [
+    'passwordPolicy',
+  ];
+
+  constructor(
+    @Host()
+    private formDir: FormGroupDirective,
+  ) {
+  }
+
+  ngOnInit() {
```

Comment:
I think we should explicity add public for all methods

Reply:
Enabled the `member-access` tslint rule.

I'll commit a fix for all classes in this PR.